book/accessibility.md

```diff
@@ -1482,10 +1486,10 @@ class LineLayout:
 ```
 
 For the [focus example](examples/example14-focus.html), the focus outline
-of an `<a>` element becomes red, as in Figure 6.
+of an `<a>` element becomes thicker and red, as in Figure 6.
```

Comment:
The red isn't visible in the book, but "thicker" is and they'll hopefully trust us on "red"